### PR TITLE
TAV-299: Fixed breaking at space problem

### PR DESCRIPTION
**Pull request by ssaleem-ee**

### CODE REVIEW

**Review by Ardesco — APPROVED** [review]

File: app/views/validationResult.scala.html [hunk 1]
```diff
@@ -87,15 +87,15 @@ <h1 class="form-title heading-large" id="pageHeading">@messages("bars.options.va
...
-                            Direct Debit - <font color="red">DISALLOWED</font><br>
+                            <span class="nobr">Direct Debit - <span class="disallowed-colour">DISALLOWED</span></span>
                         }
 
                         @if(metadata.isDirectCreditSupported) {
-                            Direct Credit - <font color="green">ALLOWED</font><br>
+                            <span class="nobr">Direct Credit - <span class="allowed-colour">ALLOWED</span></span>
                         } else {
-                            Direct Credit - <font color="red">DISALLOWED</font><br>
+                            <span class="nobr">Direct Credit - <span class="disallowed-colour">DISALLOWED</span></span>
```
Comment (Ardesco) [inline app/views/validationResult.scala.html:98]: Where is the closing `</td>`?

File: app/views/validationResult.scala.html [hunk 2]
```diff
@@ -87,15 +87,15 @@ <h1 class="form-title heading-large" id="pageHeading">@messages("bars.options.va
                     <tr><td align="right">@messages("bars.metadataResults.transactionTypes")</td>
                         <td align="center">
                         @if(metadata.isDirectDebitSupported) {
-                            Direct Debit - <font color="green">ALLOWED</font><br>
+                            <span class="nobr">Direct Debit - <span class="allowed-colour">ALLOWED</span></span>
                         } else {
-                            Direct Debit - <font color="red">DISALLOWED</font><br>
+                            <span class="nobr">Direct Debit - <span class="disallowed-colour">DISALLOWED</span></span>
                         }
```
Comment (Ardesco) [inline app/views/validationResult.scala.html:94]: Would it be better to have 
```
<span class="nobr">Direct Debit - 
@if(metadata.isDirectDebitSupported) {
<span class="allowed-colour">ALLOWED</span>
} else {
<span class="disallowed-colour">DISALLOWED</span>
}
</span>
```

File: app/views/validationResult.scala.html (hunk 1 above)
Comment (Ardesco) [inline app/views/validationResult.scala.html:98]: also same query for logic as per above

**Review by Ardesco — CHANGES_REQUESTED** [review]

File: app/views/validationResult.scala.html (hunk 2 above)
Comment (ssaleem-ee) [reply app/views/validationResult.scala.html:96]: 👍

**Review by Ardesco — APPROVED** [review]

**Pull request merged**